lib/ansible/modules/system/lvol.py

```diff
@@ -507,9 +532,17 @@ def main():
         else:
             # resize LV based on absolute values
             tool = None
-            if int(size) > this_lv['size']:
-                tool = module.get_bin_path("lvextend", required=True)
-            elif shrink and int(size) < this_lv['size']:
+            if size is not None:
+                if int(size) > this_lv['size']:
```

Comment:
Akasurde:
Just a heads-up - there is already a issue related to size typecasting - #32886